src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -234,15 +231,52 @@ public void run() {
 	private void startOngoingPhase() {
 		distributePlayersPerTeam();
 
-		_phase.next();
+		_phase = _phase.next();
 		
 		startDay();
 	}
 	
 	private void startFinishedPhase(PrisonEscapeTeam winnerTeam) {
-		_phase.next();
+		_phase = _phase.next();		
+		
+		boolean prisionersWon = winnerTeam.getName().equals(_prisionersTeam.getName());
+		int playersInPrison = _prisionersTeam.countArrestedPlayers();
+		
+		for (PrisonEscapePlayer player : _playersOnLobby) {
+			String language = MessageLanguageManager.getPlayerLanguage(player.getName());
```

Comment:
maybe we should make a function to get the MessageLanguageManager, so we don't have to repeat these 2 lines evrey time

Reply:
(aka get languange, and the get message manager)